metrics_utility/automation_controller_billing/collectors.py

```diff
@@ -540,6 +539,36 @@ def main_host_table(since, full_path, until, **kwargs):
     )
 
 
+@register('main_host_daily', '1.0', format='csv', description=_('Inventory data'), fnc_slicing=daily_slicing)
+def main_host_daily(since, full_path, until, **kwargs):
+    if 'main_host_daily' not in get_optional_collectors():
+        return None
+
+    return main_host_shared(
+        since,
+        full_path,
+        until,
+        f"""
+        enabled='t'
+        AND (
+            (main_host.created >= '{since.isoformat()}' AND main_host.created < '{until.isoformat()}')
+            OR
+            (main_host.modified >= '{since.isoformat()}' AND main_host.modified < '{until.isoformat()}')
+            OR
+            (main_unifiedjob.created >= '{since.isoformat()}' AND main_unifiedjob.created < '{until.isoformat()}')
```

Comment:
Should we add another OR with main_unifiedjob.finished so we got also all hosts that were used in jobs? Because we are collecting jobs data at the finish of the job, not at the job created.